bee-node/bee-node/src/workers.rs

```diff
@@ -61,7 +61,9 @@ impl<N: Node> Worker<N> for MetricsRegistryWorker {
                 ShutdownStream::new(shutdown, IntervalStream::new(interval(PROCESS_METRICS_UPDATE_INTERVAL)));
 
             while ticker.next().await.is_some() {
-                process_metrics.update().await;
+                if let Err(e) = process_metrics.update().await {
+                    debug!("Cannot update process metrics: {e}.");
```

Comment:
Just checking, is this important that we are aware of this error ? Because debug is so spammed that we could barely see it

Reply:
nah. If it happens the metric won't update. We just need to catch it once in case we want to know what went wrong,

Reply:
I decided to emit the log if it fails the first time just to check that the command can actually run and ignore the rest.